free-courses-vi.md

```diff
@@ -68,8 +69,12 @@
 ### C
 
 * [Bài toán kinh điển trong lập trình C++](https://www.youtube.com/watch?v=2SsyOv0koNE&list=PL33lvabfss1zRuwxONgKLc_BBsZ-Y2B6b) - Kteam
+* [Khóa học lập trình C](https://nguyenvanhieu.vn/khoa-hoc-lap-trinh-c/) - Lập Trình Không Khó
+* [Khóa học hướng đối tượng](https://nguyenvanhieu.vn/series/huong-doi-tuong-cpp/) - Lập Trình Không Khó
 * [Học Lập Trình C++](https://www.youtube.com/watch?v=jrn6bXC6sTU&list=PLyiioioEJSxHVTaeL-ELYy6Io-I8diIVZ) - DayNhauHoc
-* [Học Lập Trình C++ - Cấu trúc dữ liệu](https://www.youtube.com/watch?v=pxwP0b77Kl0&list=PLyiioioEJSxHr-4yQvc6biuGsiYqPq35F) - DayNhauHoc
+* [Học Lập Trình C++ - Cấu trúc dữ liệu](https://www.youtube.com/watch?v=pxwP0b77Kl0&list=PLyiioioEJSxHr-4yQvc6biuGsiYqPq35F) - 
```

Comment:
for Youtube, we prefer the "playlist" links, for example https://www.youtube.com/playlist?list=PL33lvabfss1xagFyyQPRcppjFKMQ7lvJM